docs/memory.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Using the Memory View
+---
+
+* toc
+{:toc}
+
+## What is it?
+
+Dynamically allocated Dart objects e.g., ```new ClassName()``` live in a portion of memory called the heap.
+
+DevTool’s Memory profile tab lets you peek at how an isolate is using memory at a given moment. This page also provides accumulator counts that you can use to study the rate of memory allocation. The accumulators are useful if you suspect that your application is leaking memory, or has other bugs relating to memory allocation.
+
+The memory view displays information about a Flutter application's memory usage as well the ability to detect memory leaks
+controlling the VMs allocation accumulator. Memory profiler consists of four parts, each increasing in granularity:
+- Memory usage overview chart
+- Event timeline
+- Classes allocated in the heap total instances, bytes allocated, accumulator of allocations since last reset
+- Instances of a class type
+
+**Use a profile build of your application to analyze performance.** Memory usages are not indicative of release
+performance unless your application is run in profile mode.
```

Comment:
Can you add a quick explanation of why? Unlike CPU profiling where debug mode stats are completely bogus, a lot of the memory stats are still ok.

Reply:
+1 to more explanation.  :)

Reply:
Use a profile build of your application to analyze performance. Memory usage is not indicative of release performance unless your application is run in profile mode. In general, memory usage is fairly accurate in relative terms in debug, release or profile mode. However, the absolute memory used maybe higher in debug versus release builds.